events/tests/TESTS/events/timing/main.cpp

```diff
@@ -128,7 +152,8 @@ void semaphore_timing_test()
 // Test setup
 utest::v1::status_t test_setup(const size_t number_of_cases)
 {
-    GREENTEA_SETUP((number_of_cases + 1)*TEST_EVENTS_TIMING_TIME / 1000, "default_auto");
+    GREENTEA_SETUP((number_of_cases + 1) * duration_cast<seconds>(TEST_EVENTS_TIMING_TIME_DURATION).count(),
```

Comment:
Looks like we had some bad style before. Is `events/tests/TESTS/events/timing` somehow exclused from `astyle` checking?

Reply:
It’s not excluded...but I just tried running astyle on it (the master branch version) and it returned “Unchanged”.  Maybe ´.astylerc’ needs updating?